{
    "_type": "pyessv._model.term", 
    "canonicalName": "qc-flag-channel-5-failure", 
    "createDate": "2018-03-09 00:00:00+00:00", 
    "data": {
        "dimension": "time", 
        "flagMeanings": "not_used good_data  bad_data_channel_failure suspect_data_time_stamp_error", 
        "flagValues": "0b,1b, 2b, 3b", 
        "longName": "Data Quality flag: Receiver channel 5 failed", 
        "name": "qc_flag_channel_5_failure", 
        "type": "byte", 
        "units": "1"
    }, 
    "idx": 5, 
    "label": "qc_flag_channel_5_failure", 
    "rawName": "qc_flag_channel_5_failure", 
    "status": "pending", 
    "uid": "f00b19a3-65dc-41ff-b314-7f0a9cd18f60"
}